docs/tools/cicd/happo.md

```diff
@@ -0,0 +1,30 @@
+# Happo 
+
+Happo is a UI diff checker tool that helps compare the UI pages from branches to master. When styling changes have been made, Happo notifies that a diff has occured and asks a reviewer to review the Happo changes before allowing the PR to be merged in.
+
+Happo is part of the Static Code Analysis process. To read more about it click [here](https://transcom.github.io/mymove-docs/docs/backend/guides/guide-to-static-analysis-security-workflow)
```

Comment:
Is this true? I don't see mention of Happo in the link.

Reply:
So I don't think Happo is part of the static code analysis. We should remove that part.

Reply:
Removed